src/Common/Config/ConfigProcessor.cpp

```diff
@@ -40,8 +47,30 @@ namespace ErrorCodes
 {
     extern const int FILE_DOESNT_EXIST;
     extern const int CANNOT_LOAD_CONFIG;
+#if USE_SSL
+    extern const int BAD_ARGUMENTS;
+#endif
 }
 
+#if USE_SSL
+namespace
+{
+
+/// Get method for string name. Throw exception for wrong name
+EncryptionMethod getEncryptionMethod(const std::string & name)
+{
+    if (name == "AES_128_GCM_SIV")
```

Comment:
I moved `getEncryptionMethod()` into Compression/CompressionCodecEncrypted.h/cpp. It's a sort of public function in DB:: namespace. I am not sure about renaming `getMethodName` into `getEncryptionMethodName`. Because in the same file Compression/CompressionCodecEncrypted.cpp we also have: `getMethodCode`, `getMethod`. Should we also rename it? Or we keep `getMethodName` as it is now?

Reply:
Ah, I commented on this one month ago already :-) I'll write another comment below.

Reply:
OK.